feature-libs/product-configurator/rulebased/core/state/actions/configurator-variant.action.ts

```diff
@@ -21,7 +21,20 @@ export class SearchVariants extends StateUtils.EntityLoadAction {
 
 export class SearchVariantsFail extends StateUtils.EntityFailAction {
   readonly type = SEARCH_VARIANTS_FAIL;
-  constructor(public payload: { ownerKey: string; error: ErrorActionType }) {
+  /**
+   * @deprecated Please use the `error` parameter with a non-null, non-undefined value.
+   *             Support for `null` or `undefined` will be removed in future versions,
+   *             along with the feature toggle `ssrStrictErrorHandlingForHttpAndNgrx`.
+   */
+  constructor(payload: { ownerKey: string; error: null | undefined });
```

Comment:
Many thanks for a careful review. Good catch!

Hmm. it looks like I didn't save and commit latest changes in this file.
Moreover, I can see `null | undefined` being used here, instead of `any` (but should be `any`).